composer.json

```diff
@@ -23,9 +23,9 @@
     "require": {
         "php": ">=5.6",
         "beatswitch/lock": "^0.2.0",
-        "illuminate/auth": "~5.3.0",
-        "illuminate/database": "~5.3.0",
-        "illuminate/support": "~5.3.0"
+        "illuminate/auth": "~5.4.0",
+        "illuminate/database": "~5.4.0",
+        "illuminate/support": "~5.4.0"
```

Comment:
If nothing breaks then perhaps it's best to add the new version as `~5.3.0|~5.4.0` so I can tag a patch release instead of a minor release.